lms/templates/ccx/grading_policy.html

```diff
@@ -3,7 +3,7 @@
 <div id="warn-coach" class="wrapper-msg urgency-high warning">
   <div class="msg">
     <i class="msg-icon icon-warning-sign"></i>
-    <div class="msg-content">
+    <div class="msg-content" tabindex="0">
```

Comment:
@sarina The CCX Test Plan is located here: https://openedx.atlassian.net/wiki/display/OPEN/CCX+Test+Plan